pkg/controller/storagecluster/reconcile.go

```diff
@@ -597,6 +614,17 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 							},
 						},
 					},
+					Env: []corev1.EnvVar{
```

Comment:
What about all the env variables in [this example](https://github.com/rook/rook/blob/release-1.4/cluster/examples/kubernetes/ceph/osd-purge.yaml#L18-L48)? Are they not needed?

Reply:
Added other variables to the code

Reply:
They will be needed